Reference documentation for the CSS Custom highlight API (#23907)
mdn/content: files/en-us/web/api/highlightregistry/size/index.md

---
title: HighlightRegistry.size
slug: Web/API/HighlightRegistry/size
page-type: web-api-instance-property
tags:
  - API
  - Property
  - CSS Custom Highlight API
  - Reference
  - size
browser-compat: api.HighlightRegistry.size
---

{{APIRef("CSS Custom Highlight API")}}

> The **`size`** property returns the number of {{domxref("Highlight")}} objects in a {{domxref("HighlightRegistry")}}.

Reviewer: ```suggestion
The **`size`** property returns the number of {{domxref("Highlight")}} objects in the {{domxref("HighlightRegistry")}}.
```

(we use "the" in the other descriptions)

## Value

> The value of `size` is a read-only integer representing how many entries the {{jsxref("Map")}}-like `HighlightRegistry` object has.

Reviewer: ```suggestion
A read-only integer indicating how many `Highlight` objects the registry contains.
```
